pkg/sync/rpc/server.go

```diff
@@ -87,14 +87,18 @@ type PurgeStatus struct {
 
 func NewPurgeStatus() *PurgeStatus {
 	return &PurgeStatus{
-		// avoid possible division by zero
-		total: 1,
+		RWMutex: sync.RWMutex{},
```

Comment:
Seems we don't really need this function?

Also, `RWMutex: sync.RWMutex{},` shouldn't be necessary. It's not a pointer.

Same below for the other statuses.

Reply:
I think we need this function, this function helps us to aggregate the usage of PurgeStatus. I will remove the RWMutex field.

Reply:
Currently, there is no other usage for this function, so we can just do `PurgeStatus: &PurgeStatus{}` below.